students/20th/suyeonkim/westagram/js/main.js

```diff
@@ -0,0 +1,131 @@
+const commentBar = document.querySelector('.feed__commentBar');
+const commentLists = document.querySelector('.feed__comments__lists');
+const commentUploadBtn = document.querySelector('.feed__comment__upload');
+
+
+// addComment
+const addComment = () => {
+    const commentList = document.createElement('li');
+    const comentPara = document.createElement('p');
+    const goodBtn = document.createElement('div');
+    const goodIcon = document.createElement('i');
+    const delBtn = document.createElement('div');
+    const delIcon = document.createElement('i');
+
+    // addComment
+    commentList.className = 'feed__comments__list';
+    comentPara.className = 'feed__comments__contents';
+    goodBtn.className = 'feed__comments__goodBtn';
+    goodIcon.className = 'fas fa-heart fa-xs';
+    delBtn.className = 'feed__comments__delBtn';
+    delIcon.className = 'fas fa-minus-circle delBtn';
+
+    commentLists.appendChild(commentList);
+    commentList.appendChild(comentPara);
+    commentList.appendChild(goodBtn);
+    goodBtn.appendChild(goodIcon);
+    commentList.appendChild(delBtn);
+    delBtn.appendChild(delIcon);
+
+    comentPara.innerText = commentBar.value;
+    commentBar.value = '';
+
+    // goodBtn
+    goodBtn.addEventListener('click', () => {
+        const changeGood = goodIcon.classList;
+
+        if(changeGood.contains('goodBtn')) {
+            changeGood.remove('goodBtn');
+        } else {
+            changeGood.add('goodBtn');
+        }
+    })
+    
+    // delBtn
+    delBtn.addEventListener('click', () => {
+        commentLists.removeChild(commentList);
+    })
+};
+
+// init
+const addCommentInit = () => {
+    commentBar.addEventListener('keydown', function(e){
+        if(e.keyCode == 13 && commentBar.value !== '') {
```

Comment:
빈 스트링은 true 값으로 처리되기 때문에
```suggestion
        if(e.keyCode == 13 && commentBar.value) {
```
로 표현할 수 있습니다.
자바스크립트에서 true, false로 처리되는 값에 대해서 한번 보시면 좋을것 같네요!